### PR TITLE
[Enhancment][Hans] add rounded corners for header

### PR DESCRIPTION
**Pull request by HansGabriel**
**Clickup Task:**
https://app.clickup.com/t/865cr4efg
https://app.clickup.com/t/865cr7z3c
https://app.clickup.com/t/865cr4e1y

**Description:**
Add rounded corners for header

**Images/Videos:**
Followed this design:
image

### CODE REVIEW

**Review by ezerssss — APPROVED** [review]

**Comment by jedmamosto** [comment]
Oof, I think you have to fix the conflicts first nong

I'm having this error: 

image

**Comment by HansGabriel** [comment]
> Oof, I think you have to fix the conflicts first nong
> 
> I'm having this error:
> 
> image

Um you have to run `npm install` first since I added new dependencies

**Review by jedmamosto — CHANGES_REQUESTED** [review]
Good job nong! However, I'd like to request some changes:

(1) Fix awkward `Sell` container - maybe have the sell container stick to the bottom? @ezerssss what do u think?

image


(2) Horizontally align `Category Picker` (category picker ang ma adjust) with `Category` + Have the category picker match font size and color with that of 'Enter product name', 'Enter product price' 

image


(3) Have `Category Picker` (category picker ma adjust) vertically align with `Set` of Meetup Time, maybe setting it to flex-end tapos margin right of 12px? + Also have the font size and color match with that of 'Enter product price' and the like.

image

**Review by ezerssss — APPROVED** [review]

**Review by caspted — CHANGES_REQUESTED** [review]
Screenshot_20230726-222409_Gallery

I request change for this in android since there is suppose to be a shadow there

File: screens/Sell/CategoryPicker.tsx [hunk 1]
```diff
@@ -40,8 +40,8 @@ export default function CategoryPicker(props: PropsInterface) {
         }}
         buttonTextAfterSelection={renderText}
         buttonTextStyle={{
-          color: buttonColor,
-          fontSize: 14,
+          color: selectedColor,
+          fontSize: 12,
```
Comment (jedmamosto) [inline screens/Sell/CategoryPicker.tsx:44]: Put it back to fontSize: 14 nong

File: screens/Sell/Sell.tsx [hunk 2]
```diff
@@ -291,17 +293,18 @@ export default function Sell({ navigation }: RootNavigationProps) {
...
                   <Text className="text-lg font-bold">
                     Preferred Meetup Time
                   </Text>
-                  <Text className="text-lg font-light text-unihub-gray-100">
-                    {time ? formatTime(time) : 'Set'}
-                  </Text>
-                </TouchableOpacity>
+                  <TouchableOpacity onPress={showDatePicker}>
+                    <Text
+                      className={`text-md px-5 py-1 font-light ${timeColor}`}
```
Comment (jedmamosto) [inline screens/Sell/Sell.tsx:303]: there's not `text-md` in tailwind ata? perhaps use `text-xs` para 14px

**Review by jedmamosto — CHANGES_REQUESTED** [review]
Last change nong then goods na man

**Comment by jedmamosto** [comment]
`text-sm` gali nong mb

**Review by jedmamosto — APPROVED** [review]
noice

**Pull request merged**